pkg/cloudprovider/types.go

```diff
@@ -215,10 +215,11 @@ func (i InstanceTypeOverhead) Total() v1.ResourceList {
 }
 
 // An Offering describes where an InstanceType is available to be used, with the expectation that its properties
-// may be tightly coupled (e.g. the availability of an instance type in some zone is scoped to a capacity type)
+// may be tightly coupled (e.g. the availability of an instance type in some zone is scoped to a capacity type) and
+// these properties are captured with labels in the Constraints map. v1beta1.CapacityTypeLabelKey and
```

Comment:
I think we should update this comment now that we changed this from a constraints map over into `scheduling.Requirements`